sdk/spring/spring-messaging-azure-eventhubs/pom.xml

```diff
@@ -109,24 +109,18 @@
       <version>4.0.0</version> <!-- {x-version-update;org.mockito:mockito-core;external_dependency} -->
       <scope>test</scope>
     </dependency>
-    <dependency>
-      <groupId>org.powermock</groupId>
-      <artifactId>powermock-api-mockito2</artifactId>
-      <version>2.0.9</version> <!-- {x-version-update;org.powermock:powermock-api-mockito2;external_dependency} -->
-      <scope>test</scope>
-    </dependency>
-    <dependency>
-      <groupId>org.powermock</groupId>
-      <artifactId>powermock-module-junit4</artifactId>
-      <version>2.0.9</version> <!-- {x-version-update;org.powermock:powermock-module-junit4;external_dependency} -->
-      <scope>test</scope>
-    </dependency>
     <dependency>
       <groupId>org.junit.jupiter</groupId>
       <artifactId>junit-jupiter-api</artifactId>
       <version>5.8.2</version> <!-- {x-version-update;org.junit.jupiter:junit-jupiter-api;external_dependency} -->
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.junit.vintage</groupId>
+      <artifactId>junit-vintage-engine</artifactId>
+      <version>5.8.2</version> <!-- {x-version-update;org.junit.vintage:junit-vintage-engine;external_dependency} -->
+      <scope>test</scope>
```

Comment:
Why are we using the vintage engine here?

Reply:
Vintage engine was added here as previously JUnit 4 was being pulled through PowerMock, the option was to either pull JUnit 4 in directly or use vintage engine from JUnit 5 which pulls JUnit 4 to be used in a JUnit 5 context (from my understanding)

Reply:
Thanks @alzimmermsft. I think that was unintentional for using the Junit 4 engine from PowerMock. I know it's not in the scope of this PR; I will create another PR to upgrade to `junit-jupiter` after this PR is merged.